script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -615,7 +615,7 @@ He said that he would savor killing me.
 [sha:fed239ac4221949b210508e73dc508786161f80f]{
 -- Page 341, Offset 32612.
 -- 「……つまらない。口上はそれで終わりか、下郎」
-"...How boring. Are you finally done blathering, serf?"
+"What dreariness. Are you finally done blathering, serf?"
```

Comment:
I read dreary as grey, dismal, depressing rather than dull, boring tedious.

Reply:
Sure, though I'd like some suggestions on how to change it regardless.